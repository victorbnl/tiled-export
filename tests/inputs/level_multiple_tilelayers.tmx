<?xml version="1.0" encoding="UTF-8"?>
<map version="1.9" tiledversion="1.9.2" orientation="orthogonal" renderorder="right-down" width="10" height="5" tilewidth="48" tileheight="48" infinite="0" backgroundcolor="#00aaff" nextlayerid="3" nextobjectid="1">
 <editorsettings>
  <export format="lua"/>
 </editorsettings>
 <tileset firstgid="1" source="tileset.tsx"/>
 <layer id="1" name="Tile Layer 1" width="10" height="5">
  <data encoding="csv">
0,0,0,0,0,0,0,2,0,0,
0,0,0,0,0,0,0,2,0,0,
0,0,0,0,0,0,0,2,0,0,
1,1,0,0,1,1,1,1,1,1,
3,3,0,0,3,3,3,3,3,3
</data>
 </layer>
 <layer id="2" name="Tile Layer 2" width="10" height="5">
  <data encoding="csv">
0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,
0,0,2,2,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0
</data>
 </layer>
</map>
