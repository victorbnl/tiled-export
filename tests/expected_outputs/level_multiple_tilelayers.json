{
  "version": "1.9",
  "tiledversion": "1.9.2",
  "class": "",
  "orientation": "orthogonal",
  "renderorder": "right-down",
  "width": 10,
  "height": 5,
  "tilewidth": 48,
  "tileheight": 48,
  "backgroundcolor": "#00aaff",
  "nextlayerid": 3,
  "nextobjectid": 1,
  "compressionlevel": -1,
  "infinite": false,
  "tilesets": [
    {
      "name": "",
      "firstgid": 1,
      "source": "tileset.tsx"
    }
  ],
  "layers": [
    {
      "x": 0,
      "y": 0,
      "offsetx": 0,
      "offsety": 0,
      "parallaxx": 1,
      "parallaxy": 1,
      "opacity": 1,
      "visible": true,
      "id": 1,
      "name": "Tile Layer 1",
      "class": "",
      "width": 10,
      "height": 5,
      "data": [
        0, 0, 0, 0, 0, 0, 0, 2, 0, 0,
        0, 0, 0, 0, 0, 0, 0, 2, 0, 0,
        0, 0, 0, 0, 0, 0, 0, 2, 0, 0,
        1, 1, 0, 0, 1, 1, 1, 1, 1, 1,
        3, 3, 0, 0, 3, 3, 3, 3, 3, 3
      ],
      "type": "tilelayer"
    },
    {
      "x": 0,
      "y": 0,
      "offsetx": 0,
      "offsety": 0,
      "parallaxx": 1,
      "parallaxy": 1,
      "opacity": 1,
      "visible": true,
      "id": 2,
      "name": "Tile Layer 2",
      "class": "",
      "width": 10,
      "height": 5,
      "data": [
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
        0, 0, 2, 2, 0, 0, 0, 0, 0, 0,
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0
      ],
      "type": "tilelayer"
    }
  ],
  "type": "map"
}
